website/content/docs/api-gateway/usage/basic-usage.mdx

```diff
@@ -0,0 +1,131 @@
+| `-sds-server-host`     | Integer value that specifies the port number for the secret discovery service (SDS).                                                                                                  | Optional | `9090`                                                                  |
+
+You can also issue the `version` command to print the Consul API Gateway version to the console:
+
+```shell-session
+$ ./consul-api-gateway version
+consul-api-gateway 0.1.0
+```
+--->
+
+## Common Error Messages
+
+Some of the errors messages commonly encountered during installation and operations of Consul API Gateway are listed below, along with suggested methods for resolving them.
+
+If the error message is not listed on this page, it may be listed on the main [Consul Common errors][consul-common-errors] page. If the error message is not listed on that page either, please consider following our general [Troubleshooting Guide][troubleshooting] or reach out to us in [Discuss](https://discuss.hashicorp.com/).
+
+<!---
+***************************************************************************
+Use markdown's Reference-Style links when including hyperlinks. This makes it easier to read the content im markdown.
+
+Each common error should have its own section on this page. Each section
+should start with a heading line that is a short description of the error or
+the text of the error.
+
+Two examples:
+    ### Failed opening file during installation
+    ### Message: "Can't connect to repository" when running Helm chart
+
+******** Template for new Error Messages ********
+ Copy and paste the following 13 lines when adding a new error message to this page.
+
+### Title for this error
+
+```
+Replace with text of example error message
+```
+**Conditions:**
+REPLACE THIS with description of when and why the error is typically seen
+
+**Impact:**
+REPLACE THIS with description of most likely impact of the event that caused this error to occur
+
+**Recommended Action:**
+REPLACE THIS with the actions the user should take to try to correct the situation
+
+***************************************************************************
+--->
+
+### Helm installation failed: "no matches for kind"
+
+```log
+Error: INSTALLATION FAILED: unable to build kubernetes objects from release manifest: [unable to recognize "": no matches for kind "GatewayClass" in version "gateway.networking.k8s.io/v1alpha2", unable to recognize "": no matches for kind "GatewayClassConfig" in version "api-gateway.consul.hashicorp.com/v1alpha1"]
+```
+**Conditions:**
+When this error occurs during the process of installing Consul API Gateway, it is usually caused by not having the required CRD files installed in Kubernetes prior to installing Consul API Gateway.
```

Comment:
```suggestion
Consul API Gateway generates this error when the required CRD files have not been installed in Kubernetes prior to installing Consul API Gateway. 
```
Trying to restate in active voice. Is this still accurate? Does this message only appear during the installation step?

Reply:
I believe it should only appear during the installation step, but @nathancoleman @andrewstucki @mikemorris can one of you confirm?